web/modules/environment_indicator/README.md

```diff
@@ -0,0 +1,163 @@
+INTRODUCTION
```

Comment:
@ccdatapdx , modules should be added via composer rather than being added to the file system. Please add this using `lando composer require drupal/environment_indicator`. The PR should only be the file change to composer.json, composer.lock, a couple of config files, and the settings to show the different colors. See the portlandor repo for an example.

Reply:
Ah yes, ok. All coming back to me.